mobile/android/java/Interface.java

```diff
@@ -1,8 +1,8 @@
 package org.iota.mobile;
 
 public class Interface {
-    public static native String iota_pow_trytes(String trytes, int mwm);
-    public static native String[] iota_pow_bundle(String[] bundle, String trunk, String branch, int mwm);
+    public static native String iota_pow_trytes(String trytes, uint8_t mwm);
```

Comment:
```suggestion
    public static native String iota_pow_trytes(String trytes, int mwm);
```

Should this stay as `int` since it's Java?

Reply:
@HowJMay please fix it, java have no unsigned integer type.